src/main.cpp

```diff
@@ -2032,8 +2032,11 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         return true;
     }
 
-    bool fScriptChecks = true;
-    if (fCheckpointsEnabled) {
+    const int64_t yesterday = GetTime() - 24 * 3600;
+    // Blocks older than 24 hours are assumed to have so much POW behind them, that checking of the scripts
+    // adds no additional value. So skip them.
+    bool fScriptChecks = block.nTime > yesterday;
```

Comment:
Shouldn't the check be for number of blocks instead?  Explanation here:

https://www.reddit.com/r/Bitcoin/comments/49v808/peter_todd_on_twitter_tldr_bitcoin_classic_is/d0v5tqy